src/mon/HealthMonitor.h

```diff
@@ -45,9 +45,10 @@ class HealthMonitor : public QuorumService
   virtual bool service_dispatch(MonOpRequestRef op);
 
   virtual void start_epoch() {
+    epoch_t epoch = get_epoch();
     for (map<int,HealthService*>::iterator it = services.begin();
          it != services.end(); ++it) {
-      it->second->start(get_epoch());
```

Comment:
@tchaikov  :smile: Thank you sir!For your review. I am not sure that whether the compiler can inline it.If we do not  statement inline in the constructor function.If that true I will close this pr.